### ISSUE TITLE
tutorials?

### ISSUE THREAD

**Issue by AJTJ** [nannou-org/nannou#666]
Hey friends,

Really enjoying playing around with Nannou, but I think we could really use some more tutorials.

The OSC tutorial on the website isn't even complete.

<3

**Comment by j6k4m8** [comment]
Huge agree here — I think the [p5.js Learn](https://p5js.org/learn/) section is a particularly good model.

If there's some (emotional) support for this, I'd be interested in contributing!

**Comment by mitchmindtree** [comment]
Absolutely, I think tutorials can be some of the most valuable contributions :)

For those stumbling upon this issue, this is the tutorials section of the guide referred to above:

https://guide.nannou.cc/tutorials.html

There are a list of ideas for future tutorials at the link, but tutorials on other topics would likely also be welcome! If you're unsure whether they would fit, feel free to open an issue first to ask or discuss first :+1:
